tests/TestCase/View/Helper/BreadcrumbsHelperTest.php

```diff
@@ -357,6 +357,8 @@ public function testInsertAfter()
      */
     public function testRender()
     {
+        $this->assertEmpty($this->breadcrumbs->render());
```

Comment:
You might want `assertSame('', $this->breadcrumbs->render())` or `assertNull()`.